tools/add_triage_signature.py

```diff
@@ -92,12 +91,22 @@ def get_events_json(cluster_url, cluster_id):
     return res.json()
 
 
-def get_cluster_logs(cluster_url: str, cluster_id: str, output_file: str):
-    res = requests.get(f"{cluster_url}/cluster_{cluster_id}_logs.tar")
-    res.raise_for_status()
+@functools.lru_cache(maxsize=100)
+def get_remote_archive(tar_url: str) -> nestedarchive.RemoteNestedArchive:
```

Comment:
I think using annotation, with out having something actually verify them is bad
They are like remarks.
They will be misleading the next time someone changes the code.

Reply:
Removed all type annotations in d82a042

Reply:
I have to disagree with you on that.
For example, 
```
- def group_similar_strings(ls: List[str], ratio: int) -> List[str]:
+ def group_similar_strings(ls, ratio):
```
I can't really know how the grouping works unless I would investigate the function logic. It could be concatenated into a single string. The `List` indication immediately guides me on how the function works.

Reply:
I don't think the benefits of type hints are in question, I believe everyone agrees on that. @ronniel1 prefers we have a discussion about whether we want them or not before introducing them in the code because they have the potential to become confusing as they grow out of sync with reality because there's nothing to keep them in check.